.gitignore

```diff
@@ -145,3 +145,6 @@ dist
 .history
 
 # End of https://www.toptal.com/developers/gitignore/api/node,go,visualstudiocode
+
+# linux binary
+/oh-my-posh3
```

Comment:
maybe add the exe as well, that way it's ignore everywhere

Reply:
`*.exe` already gets ignored right?  Line 7

Reply:
Yup, you're right!